js/Application.js

```diff
@@ -96,6 +96,7 @@ define(
 				});
 				this.companyInfoTemplate = config.companyInfoCustomHtmlTemplate;
 				this.showCompanyInfo = config.showCompanyInfo;
+				this.isLoggedIn = ko.computed(() => authApi.isAuthenticated());
```

Comment:
Can you make this ko.pureComputed().  ko.computed is more heavy weight, and requires cleanup after component is disposed.  The pureComputed() is appropraite if there are no 'side effects' in the evaluation of the computed (ie: you set some other observable state in the process of evaluated the computed expression).  Since that's all you are doing here, I think pureComputed is appopriate.

Reply:
Thanks! Done.